shared/common-adapters/icon.desktop.js

```diff
@@ -84,7 +84,7 @@ export default class Icon extends Component {
       return <FontIcon
         title={this.props.hint}
         style={{...styles.icon, ...this.props.style}}
-        className={`fa ${iconType}`}
+        className={`fa ${iconType} ${this.props.className}`}
```

Comment:
we should talk about generally plumbing all these react base props through, maybe some utility